### PR TITLE
feat: full implementation of contrast themes (resolves #495, resolves #840, resolves #952)

### LINKED ISSUE #341: Design system / typography

### LINKED ISSUE #342: Design system / colours

### LINKED ISSUE #345: Design system / dividers

### LINKED ISSUE #346: Design system / notifications

### LINKED ISSUE #348: Design system / checkboxes

### LINKED ISSUE #349: Design system / radio buttons

### LINKED ISSUE #350: Design system / text inputs

### LINKED ISSUE #352: Design system / search inputs

### LINKED ISSUE #353: Design system / textareas

### LINKED ISSUE #354: Design system / selects

### LINKED ISSUE #356: Design system / field error messages

### LINKED ISSUE #359: Design system / borderless buttons

### LINKED ISSUE #363: Design system / support bar

### LINKED ISSUE #364: Design system / top navigation

### LINKED ISSUE #365: Design system / footer

### LINKED ISSUE #367: Design system / accordions

### LINKED ISSUE #371: Design system / cards: resource

### LINKED ISSUE #414: Design system / cards: resource collection

### LINKED ISSUE #495: Adjustable contrast

### LINKED ISSUE #840: Settings - Website accessibility preferences: image labels should be improved.
On the contrast adjustments, the screen reader reads "Image" for all the images associated with each option. Better labels could be provided for these images.

### LINKED ISSUE #952: Add the contrast options to the top navigation, so user can adjust the contrast even without an account and while browsing home pages.
Head tracker user had difficulty seeing the buttons in the default colour while creating an account. Once they created their account, they were not able to find the display preferences settings on their own. Having the contrast option up front will greatly help the navigation.

### PR DESCRIPTION
**Pull request by greatislander**
Fully implements contrast themes.

- [x] Resolves #341
- [x] Resolves #342
- [x] Resolves #345
- [x] Resolves #346
- [x] Resolves #348
- [x] Resolves #349
- [x] Resolves #350
- [x] Resolves #352
- [x] Resolves #353
- [x] Resolves #354
- [x] Resolves #356
- [x] Resolves #359
- [x] Resolves #363
- [x] Resolves #364
- [x] Resolves #365
- [x] Resolves #367
- [x] Resolves #371
- [x] Resolves #414
- [x] Resolves #495
- [x] Resolves #840
- [x] Resolves #952

### Prerequisites

If this PR changes PHP code or dependencies:

- [x] I've run `composer format` and fixed any code formatting issues.
- [x] I've run `composer analyze` and addressed any static analysis issues.
- [x] I've run `php artisan test` and ensured that all tests pass.
- [x] I've run `composer localize` to update localization source files and committed any changes.

If this PR changes CSS or JavaScript code or dependencies:

- [x] I've run `npm run lint` and fixed any linting issues.
- [x] I've run `npm run build` and ensured that CSS and JavaScript assets can be compiled.

### CODE REVIEW

**Comment by jobara** [comment]
@greatislander I tried the theme selector dropdown with VoiceOver, and it worked well.

**Comment by greatislander** [comment]
For testing:

1. Guest (without an account) should be able to choose a theme from the menu, and it should be saved in their browser.
2. If guest registers an account, the theme they have chosen should be saved to their account (and should show up in their website accessibility preferences).
3. If a user changes their theme, it should persist after they log out.

**Review by cherylhjli — APPROVED** [review]

File: app/Actions/Fortify/CreateNewUser.php [hunk 1]
```diff
@@ -75,8 +75,8 @@ public function create(array $input): User
             ]
         )->validate();
 
-        Cookie::queue('theme', 'light');
         Cookie::queue('locale', $input['locale']);
+        Cookie::queue('theme', Cookie::get('theme', 'system'));
```
Comment (jobara) [inline app/Actions/Fortify/CreateNewUser.php:79]: Does this actually need to be queued. Shouldn't it already be set if you are getting it?

Comment (jobara) [reply app/Actions/Fortify/CreateNewUser.php:79]: I see, you're queueing it in case there was no cookie found and it used the fallback value.

**Pull request merged**